### PR TITLE
Update game.sh

### PR DESCRIPTION
**Pull request by JoseEscauriza**
Added line 10

### CODE REVIEW

File: game.sh [hunk 1]
```diff
@@ -2,3 +2,9 @@ echo "Hello from `whoami`"
 echo "Terminal time:`date`"
 echo "This line should be here"
 echo "Spider locomotion"
+
```
Comment (Eyongkevin) [inline game.sh:5]: Hey, you introduced a lot empty new lines. Can you fix it please.

File: game.sh [hunk 2]
```diff
@@ -2,3 +2,9 @@ echo "Hello from `whoami`"
...
+
+
+
+
+echo "New line from Jose E"
```
Comment (Eyongkevin) [inline game.sh:10]: Thank you, good work

**Review by Eyongkevin — CHANGES_REQUESTED** [review]

File: game.sh (hunk 1 above)
Comment (JoseEscauriza) [reply game.sh:5]: On it boss

**Comment by JoseEscauriza** [comment]
Deleted empty spaces, added line 5

**Pull request closed without merging**